.editorconfig

```diff
@@ -0,0 +1,33 @@
+# EditorConfig is awesome: http://EditorConfig.org
+
+# top-most EditorConfig file
+root = true
+
+# Unix-style newlines with a newline ending every file
+[*]
+end_of_line = lf
+insert_final_newline = true
+
+# Matches multiple files with brace expansion notation
+# Set default charset
+[*.{js,py}]
+charset = utf-8
+
+# 4 space indentation
+[*.py]
+indent_style = space
+indent_size = 4
+
+# Tab indentation (no size specified)
+[Makefile]
+indent_style = tab
+
+# Indentation override for all JS under lib directory
```

Comment:
We can remove that. No JavaScript in the repo. And working with Elm, probably no JavaScript ever ; )

**Or** we can change that to work for Elm (4 spaces instead of 2):

```yaml
[**.elm]
indent_style = space
indent_size = 4
```